recirq/quantum_chess/experiments/circuits/4gt13_92.qasm

```diff
@@ -0,0 +1,70 @@
+OPENQASM 2.0;
```

Comment:
Can we add a README.md to this directory to explain what all of these are?  I am not sure I can tell what 4gt13_92 means.

Reply:
It's not well documented anywhere I can find, and I'm not much for guessing names or reverse engineering qasm either :) I think what's most important is the size of the circuits so I made sure to document that